packages/storybook8/stories/Components/Participantitem/index.stories.tsx

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { ParticipantItem as ParticipantItemComponent } from '@azure/communication-react';
+import { Meta } from '@storybook/react';
+import { controlsToAdd, hiddenControl } from '../../controlsUtils';
+import { CustomAvatarExample } from './snippets/CustomAvatar.snippet';
+import { CustomIconExample } from './snippets/CustomIcon.snippet';
+import { ParticipantItemExample } from './snippets/ParticipantItem.snippet';
+export { ParticipantItem } from './ParticipantItem.story';
+
+export const ParticipantItemSnippetDocsOnly = {
+  render: ParticipantItemExample
+};
+
+export const CustomAvatarSnippetDocsOnly = {
+  render: CustomAvatarExample
+};
+
+export const CustomIconSnippetDocsOnly = {
+  render: CustomIconExample
+};
+
+const meta: Meta = {
+  title: 'Components/Participant Item',
+  component: ParticipantItemComponent,
+  argTypes: {
+    displayName: { ...controlsToAdd.displayName },
```

Comment:
is there a reason to use different syntax here `displayName: { ...controlsToAdd.displayName }` than what we are using everywhere else `displayName: controlsToAdd.displayName`

Reply:
Updated so pattern matches the other pages